src/bp/core/services/cms.ts

```diff
@@ -597,24 +597,26 @@ export class CMSService implements IDisposeOnExit {
     }, {})
   }
 
-  async renderElement(contentId, args, eventDestination: IO.EventDestination) {
+  async renderElement(contentId: string, args, eventDestination: IO.EventDestination) {
     const { botId, channel } = eventDestination
     contentId = contentId.replace(/^#?/i, '')
     let contentTypeRenderer: ContentType
 
+    const translateFormData = async (formData: object): Promise<object> => {
```

Comment:
why is this a closure func ? I mean this is a class, you could set this a sa private function of the calss

Reply:
I could, but it would make a method with 4 parameters and 2 lines of code. Wouldn't make it cleaner

Reply:
as you wish my friend. this is javascript, you can do whatever you want